### ISSUE TITLE
Help navigation menu is different from the main navigation menu

### ISSUE THREAD

**Issue by cechticky** [pnp-software/cordetfw_editor#51]
Using the same navigation menu for all views would improve look&feel of the tool.

![image](https://user-images.githubusercontent.com/5507223/132814504-3e6cfd2f-fa0b-4c80-8d64-54c66e7b148a.png)

vs.

![image](https://user-images.githubusercontent.com/5507223/132814540-01c9b635-2c46-4668-b1c3-80da9b59d6a9.png)

**Comment by pasetti** [comment]
The help.html page still has the "old style" which I introduced when I started the Cordet Editor project. Most other pages have been ported to a new style by Pascal. At this stage, Pascal is the only one who can port the Help page to the new style. I would therefore assign this ticket to him.
Additionally, I would ask him to fill in the section on `Update css` on the `ReadME` file by explaining how the new style pages are designed.
